EPlast/EPlast/wwwroot/scripts/app.js

```diff
@@ -0,0 +1,6 @@
+function sayHello() {
+    const compiler = document.getElementById("compiler").value;
+    const framework = document.getElementById("framework").value;
+    return `Hello from ${compiler} and ${framework}!`;
+}
+//# sourceMappingURL=app.js.map
```

Comment:
there shouldn't be this file under this location EPlast/EPlast/wwwroot/scripts/app.js, remove it please